synapse/storage/engines/sqlite.py

```diff
@@ -120,6 +120,25 @@ def attempt_to_set_isolation_level(
         # All transactions are SERIALIZABLE by default in sqlite
         pass
 
+    @staticmethod
+    def executescript(cursor: sqlite3.Cursor, script: str) -> None:
```

Comment:
This seems to be a lie, because the cursor is actually a `LoggingTransaction` that wraps a sqlite3.Cursor.

I refrained from making further type annotations here.